### DISCUSSION TITLE
suggestion missions

### DISCUSSION THREAD

**Discussion by brasilcraft54** [BG-Software-LLC/SuperiorSkyblock2#544] [Implemented Feature Requests] [answered]
missions to eat, craft, plant, and harvest.

**Comment by OmerBenGera** [comment, marked as answer]
> missions to eat, craft, plant, and harvest.

Eat - pretty useless, dont yoi think?
Plant - only planet? If so, can be done using BlocksMission
Harvest - Can be done using BlocksMission
